CHANGES.rst

```diff
@@ -40,6 +40,8 @@ Improvements
   (:pr:`4817`)
 - Failed login attempts now trigger rate limiting to prevent brute-force attacks
   (:issue:`1550`, :pr:`4817`)
+- iCalendar exports include more information and use new serializers
```

Comment:
I'd reword the changelog entry a bit. "and use new serializers" is not relevant for the changelog (that's purely internal refactoring), and I'd mention at least slightly more details on what the actual (user-facing) improvements to the ical exports are